gtsfm/frontend/verifier/ransac.py

```diff
@@ -100,7 +100,7 @@ def verify(
                 uv_norm_i1[match_indices[:, 0]],
                 uv_norm_i2[match_indices[:, 1]],
                 K,
-                method=cv2.RANSAC,
+                method=cv2.USAC_ACCURATE,
```

Comment:
have we studied if USAC has any benefit over RANSAC, from the metrics?

Reply:
I had to switch to USAC because the RANSAC in opencv >=4.5.0 leads to failures in even the simple scene unit test in verification. 

I will study the metrics more to understand this and update this PR.

Reply:
added the metrics

Reply:
thanks. if we can open an issue about this on opencv (with a simple unit test we give them that fails as a minimal reproducible example) that would be great